clar2wasm/tests/bin_tests.rs

```diff
@@ -18,6 +18,11 @@ fn test_clar2wasm_with_input() {
         if file.path().extension().unwrap_or(OsStr::new("")) != "clar" {
             continue;
         }
+        if file.path().file_name().unwrap() == "bns.clar" {
+            // bns.clar uses `block-height` which is not supported in clarity 3
+            // the clar2wasm bin should accept a ClarityVersion flag
```

Comment:
Is there an issue for this? We should have one.

Reply:
It's possible that @csgui already fixed it but I'm not sure

Reply:
Not yet. This issue has been added to the list of [known issues](https://github.com/stacks-network/clarity-wasm/issues/310#issuecomment-2146927314) affecting boot contracts.

Reply:
More than an issue affecting the boot contracts, it looks like a feature request. But an urgent one due to #310 for sure. It should have its own issue.